system/Exceptions/CastException.php

```diff
@@ -43,4 +43,20 @@ public static function forInvalidJsonFormatException(int $error)
 				return new static(lang('Cast.jsonErrorUnknown'));
 		}
 	}
+
+	public static function missingInterface($class) : self
+	{
+		return new static(lang('Cast.abstractCastMissing', [$class]));
+	}
+
+	public static function invalidCastMethod() : self
+	{
+		return new static(lang('Cast.wrongCastMethod'));
+	}
+
+	public static function invalidTimestamp() : self
+	{
+		return new static(lang('Cast.invalidTimestamp'));
+	}
```

Comment:
Framework's way of declaring new static methods is `for` + the statement. So in this case it would be `forInvalidTimestamp` etc,

Reply:
I'm sorry I wasn't clear enough. What I meant is that all new methods would begin with `for`, not only `invalidTimestamp`.

Reply:
Fixed